tests/integration/test_events.py

```diff
@@ -1236,6 +1239,79 @@ def get_message(queue_url):
         # clean up
         self.cleanup(TEST_EVENT_BUS_NAME, rule_name, target_id, queue_url=queue_url)
 
+    @pytest.mark.parametrize(
+        "schedule_expression", ["rate(1 minute)", "rate(1 day)", "rate(1 hour)"]
+    )
+    @pytest.mark.aws_validated
+    def test_create_rule_with_one_unit_in_singular_should_succeed(
+        self, events_client, schedule_expression
+    ):
+        rule_name = f"rule-{short_uid()}"
+
+        # rule should be creatable with given expression
+        try:
+            events_client.put_rule(Name=rule_name, ScheduleExpression=schedule_expression)
+        finally:
+            self.cleanup(rule_name=rule_name, events_client=events_client)
+
+    @pytest.mark.parametrize(
+        "schedule_expression", ["rate(1 minutes)", "rate(1 days)", "rate(1 hours)"]
+    )
+    @pytest.mark.aws_validated
+    @pytest.mark.xfail
+    def test_create_rule_with_one_unit_in_plural_should_fail(
+        self, events_client, schedule_expression
+    ):
+        rule_name = f"rule-{short_uid()}"
+
+        # rule should not be creatable with given expression
+        with pytest.raises(ClientError):
+            events_client.put_rule(Name=rule_name, ScheduleExpression=schedule_expression)
+
+    @pytest.mark.aws_validated
+    @pytest.mark.xfail
+    def test_verify_rule_event_content(self, events_client, logs_client):
+        log_group_name = f"/aws/events/testLogGroup-{short_uid()}"
+        logs_client.create_log_group(logGroupName=log_group_name)
+
+        log_groups = logs_client.describe_log_groups(logGroupNamePrefix=log_group_name)
+        assert len(log_groups["logGroups"]) == 1
+        log_group = log_groups["logGroups"][0]
+
+        log_group_arn = log_group["arn"]
+
+        rule_name = f"rule-{short_uid()}"
```

Comment:
nit: putting these generated names in the front might be a bit clearer and is usually what the other tests are doing as well. Kind of a more implicit pattern we have in these tests